qml/DynamicDisplayStatisticsDialog.qml

```diff
@@ -32,10 +32,13 @@ Dialog {
     y: (parent.height - height) / 2
 
     property bool targetExists: false
-
     property bool activeOk: true
+    property var availableStatisticKinds: []
 
     Component.onCompleted: {
+        // Get the available statistic kinds from the backend
+        availableStatisticKinds = controller.get_statistic_kinds()
+
         if (dataKind == "FASTDDS_LATENCY" |
                 dataKind == "NETWORK_LATENCY" |
                 dataKind == "RTPS_PACKETS_SENT" |
```

Comment:
Just a comment:
These kinds should come from backend call: `get_data_supported_entity_kinds`